src/status.rs

```diff
@@ -375,14 +395,15 @@ impl Deserialize for Status {
                     state: process_map_field!(state, "state"),
                     sensors: sensors.unwrap_or(None),
                     ext_versions: ext_versions.unwrap_or(None),
+                    extensions: BTreeMap::new(),//process_map_field!(extensions, "extensions"),
```

Comment:
Oh, wait, this isn't actually finished... I probably still have some of the commits on my laptop :laugh: